src/components/InstallButton.js

```diff
@@ -137,6 +137,13 @@ const InstallButton = ({ quickstart, location, ...props }) => {
       options.domain = 'newrelic.com';
     }
 
+    const startTarget = btoa(
+      JSON.stringify({
+        source: 'nrio',
+        id: quickstart.id,
+      })
+    );
+    Cookies.set('start_target', startTarget, options);
     Cookies.set('newrelic-quickstart-id', quickstart.id, options);
```

Comment:
I'm guessing the plan is to deprecate this cookie eventually. Maybe we could toss a comment on there like this
```suggestion
    Cookies.set('newrelic-quickstart-id', quickstart.id, options); // TODO: Remove this once the product has moved over to the new cookie above```